post_installation_configuration/configuring-multi-arch-compute-machines/creating-multi-arch-compute-nodes-azure.adoc

```diff
@@ -6,9 +6,13 @@ include::_attributes/common-attributes.adoc[]
 
 toc::[]
 
-To deploy an Azure cluster with multi-architecture compute machines, you must first create a single-architecture Azure installer-provisioned cluster that uses the multi-architecture installer binary. For more information on Azure installations, see xref:../../installing/installing_azure/installing-azure-customizations.adoc#installing-azure-customizations[Installing a cluster on Azure with customizations]. You can then add an ARM64 compute machine set to your cluster to create a cluster with multi-architecture compute machines.
+To deploy an Azure cluster with multi-architecture compute machines, you must first create a single-architecture Azure installer-provisioned cluster that uses the multi-architecture installer binary. For more information on Azure installations, see xref:../../installing/installing_azure/installing-azure-customizations.adoc#installing-azure-customizations[Installing a cluster on Azure with customizations]. 
 
-The following procedures explain how to generate an ARM64 boot image and create an Azure compute machine set that uses the ARM64 boot image. This adds ARM64 compute nodes to your cluster and deploys the amount of ARM64 virtual machines (VM) that you need.
+You can also migrate your current cluster with single-architecture compute machines to a cluster with multi-architecture compute machines. For more information, see xref:../../updating/updating_a_cluster/migrating-to-multi-payload.adoc#migrating-to-multi-payload[Migrating to a cluster with multi-architecture compute machines].
+
+After creating a multi-architecture cluster, you can add nodes with different architectures to the cluster. 
+
+For a cluster with 64-bit x86 compute machines, the following procedures explain how to generate a 64-bit ARM boot image and create an Azure compute machine set that uses the 64-bit ARM boot image. You can add compute nodes to your cluster and deploy the 64-bit ARM or 64-bit x86 virtual machines (VM).
```

Comment:
@sr1kar99 I think we can remove this entire line.

Reply:
Updated as per the suggestion.